src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -36,11 +45,15 @@ public void teleopInitialize() {
 	 * This function is called periodically during operator control
 	 */
 	@Override
-	public void teleopExecute() {}
+	public void teleopExecute() {
+		RobotMap.Component.teensyStick.button1.whenPressed(teleopCommand);
```

Comment:
This can go in initialize, and should eventually migrate to DefaultOperator's bindCommands.